src/sentry/integrations/github/integration.py

```diff
@@ -591,6 +593,7 @@ def dispatch(self, request: HttpRequest, pipeline: Pipeline) -> HttpResponseBase
                     error_short=GitHubInstallationError.MISSING_LOGIN,
                 )
 
+            pipeline.bind_state("user_access_token", payload["access_token"])
```

Comment:
is this something you added that you need later?

Reply:
i'm confused, isn't this unused? i don't see other references to `user_access_token` in this PR. you're currently passing `access_token` into `GithubSetupApiClient`

Reply:
nono, you're right, this was meant to be deleted :oopsie: